.editorconfig

```diff
@@ -193,8 +193,8 @@ dotnet_naming_rule.instance_fields_should_be_camel_case.style = instance_field_s
 
 dotnet_naming_symbols.instance_fields.applicable_kinds = field
 
-dotnet_naming_style.instance_field_style.capitalization = camel_case
-dotnet_naming_style.instance_field_style.required_prefix = _
+dotnet_naming_style.instance_field_style.capitalization = pascal_case
+dotnet_naming_style.instance_field_style.required_prefix =
```

Comment:
What's happened here?

Reply:
Indeed. No pascal case

Reply:
I changed it to get the IDE to shut up about it's "suggestions" to rename the fields. My IDE was suggesting to renaming _all_ properties to use camel case and a prefix. If you can somehow tweak the rules to only apply the prefix and camel case to the _private_ properties, then be my guest, i won't invest time into finding the correct settings to use atm.

Reply:
And yes, I can remove the commit that changed these settings from this PR, but it's staying for now, so i can keep my sanity… for now.